### ISSUE TITLE
set_php_version() raise requests.exceptions.HTTPError:

### ISSUE THREAD

**Issue by marynaperesypkina** [pmatias/python-acquia-cloud-2#2]
```
environment_id = application.environments()[env]['id']
environment = application.environment(environment_id)
environment.set_php_version(version)
```

This code raise an error. The work url for me is `https://cloud.acquia.com/api/environments/<environment_id>`

```Traceback (most recent call last):
  File "/home/user/projects/chalice/venv/lib/python3.5/site-packages/chalice/app.py", line 659, in _get_view_function_response
    response = view_function(**function_args)
  File "/home/user/projects/chalice/chalice/app.py", line 533, in queue_php_version
    queue_php_version_background(payload, object)
  File "/home/user/projects/chalice/venv/lib/python3.5/site-packages/chalice/app.py", line 915, in __call__
    return self.func(event, context)
  File "/home/user/projects/chalice/chalice/app.py", line 489, in queue_php_version_background
    util.set_php_version(site, env, version)
  File "/home/user/projects/chalice/chalice/chalicelib/util.py", line 217, in set_php_version
    environment.set_php_version(version)
  File "/home/user/projects/chalice/venv/lib/python3.5/site-packages/acapi2/resources/environment.py", line 72, in set_php_version
    return self.configure(data)
  File "/home/user/projects/chalice/venv/lib/python3.5/site-packages/acapi2/resources/environment.py", line 22, in configure
    return self.request(uri=self.uri, method="PUT", data=data)
  File "/home/user/projects/chalice/venv/lib/python3.5/site-packages/acapi2/resources/acquiadata.py", line 102, in request
    raise response.raise_for_status()
  File "/home/user/projects/chalice/venv/lib/python3.5/site-packages/requests/models.py", line 862, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://cloud.acquia.com/api/applications/<application_id>/<environment_id>
```

**Comment by pmatias** [comment]
Hi Maryna, could you try this?

```
environment = application.environments()[env]
environment.set_php_version(version)
```

**Comment by marynaperesypkina** [comment]
In this case environment is a dict
```
    environment = application.environments()[env]
    environment.set_php_version(version)
```
 

```
Traceback (most recent call last):
  File "/home/user/chalice/venv/lib/python3.5/site-packages/chalice/app.py", line 659, in _get_view_function_response
    response = view_function(**function_args)
  File "/home/user/chalice/chalice/app.py", line 533, in queue_php_version
    queue_php_version_background(payload, object)
  File "/home/user/chalice/venv/lib/python3.5/site-packages/chalice/app.py", line 915, in __call__
    return self.func(event, context)
  File "/home/user/chalice/chalice/app.py", line 489, in queue_php_version_background
    util.set_php_version(site, env, version)
  File "/home/user/chalice/chalice/chalicelib/util.py", line 214, in set_php_version
    environment.set_php_version(version)
AttributeError: 'dict' object has no attribute 'set_php_version
```

**Comment by pmatias** [comment]
ok, I'll take a look. Thanks for reporting it :)

**Comment by pmatias** [comment]
@marynaperesypkina, the bug you reported makes sense, but at the moment the Acquia API v2 is in a very alpha state and we're expecting deep changes on their side, so making changes to the client makes little sense to me right now. I'm aiming to only _add_ functional methods here and there to get the essential things working.

However, and in the meantime, I believe this should fix your current issue:

```
acquia_api = Acquia(api_key, api_secret)
acquia.environment(environment_uuid).set_php_version(version)
```

Basically, instead of getting the `application` instance, then getting the environments, to finally set the the php version, you will be just instancing the environment itself and set the php version at the same time.

I just tested it locally (switching from `7` to `5.6` and then back to `7`) and it worked as expected.

Let me know how it goes.

**Comment by pmatias** [comment]
I've spoke with Maryna, looks like the previous solution worked for her. Closing.
